web-ui/src/backup_account/backup_email/backup_email.js

```diff
@@ -39,16 +40,23 @@ export class BackupEmail extends React.Component {
     });
   }
 
+  submitHandler = (event) => {
+    event.preventDefault();
+    if (typeof this.props.onSubmit === 'function') {
```

Comment:
Is this if necessary if we define that onSubmit is required (as you said it was required in the propTypes? I think we should avoid it.